app/scripts/controllers/metamask-notifications/constants/notification-schema.ts

```diff
@@ -156,7 +151,7 @@ export const TRIGGERS: Record<OnChainTriggerTypes, Trigger> = {
   [TRIGGER_TYPES.LIDO_WITHDRAWAL_COMPLETED]: {
     supported_chains: [NOTIFICATION_CHAINS.ETHEREUM],
   },
-  [TRIGGER_TYPES.LIDO_STAKE_READY_TO_BE_WITHDRAWN]: {
-    supported_chains: [NOTIFICATION_CHAINS.ETHEREUM],
-  },
+  // [TRIGGER_TYPES.LIDO_STAKE_READY_TO_BE_WITHDRAWN]: {
+  //   supported_chains: [NOTIFICATION_CHAINS.ETHEREUM],
+  // },
```

Comment:
Remove commented out code.

As discussed this notification trigger does not exist (it is polyfilled on our service correct?)

Reply:
Done!